includes/admin/class-sensei-plugins-installation.php

```diff
@@ -0,0 +1,421 @@
+			$error     = false;
+
+			// See if the plugin is installed already.
+			if ( isset( $installed_plugins[ $plugin_file ] ) ) {
+				$installed = true;
+				$activate  = ! is_plugin_active( $installed_plugins[ $plugin_file ] );
+			}
+
+			// Install this thing!
+			if ( ! $installed ) {
+				// Suppress feedback.
+				ob_start();
+
+				try {
+					$plugin_information = plugins_api(
+						'plugin_information',
+						[
+							'slug'   => $plugin_slug,
+							'fields' => [
+								'short_description' => false,
+								'sections'          => false,
+								'requires'          => false,
+								'rating'            => false,
+								'ratings'           => false,
+								'downloaded'        => false,
+								'last_updated'      => false,
+								'added'             => false,
+								'tags'              => false,
+								'homepage'          => false,
+								'donate_link'       => false,
+								'author_profile'    => false,
+								'author'            => false,
+							],
+						]
+					);
+
+					if ( is_wp_error( $plugin_information ) ) {
+						throw new Exception( $this->get_error_message( $plugin_information ) );
+					}
+
+					$package  = $plugin_information->download_link;
+					$download = $upgrader->download_package( $package );
+
+					if ( is_wp_error( $download ) ) {
+						throw new Exception( $this->get_error_message( $download ) );
+					}
+
+					$working_dir = $upgrader->unpack_package( $download, true );
+
+					if ( is_wp_error( $working_dir ) ) {
+						throw new Exception( $this->get_error_message( $working_dir ) );
+					}
+
+					$result = $upgrader->install_package(
+						[
+							'source'                      => $working_dir,
+							'destination'                 => WP_PLUGIN_DIR,
+							'clear_destination'           => false,
+							'abort_if_destination_exists' => false,
+							'clear_working'               => true,
+							'hook_extra'                  => [
+								'type'   => 'plugin',
+								'action' => 'install',
+							],
+						]
+					);
+
+					if ( is_wp_error( $result ) ) {
+						throw new Exception( $this->get_error_message( $result ) );
+					}
+
+					$activate = true;
+
+				} catch ( Exception $e ) {
+					$error   = true;
+					$message = sprintf(
+						// translators: 1: plugin name, 2: error message.
+						__( '%1$s could not be installed (%2$s).', 'sensei-lms' ),
+						$plugin_title,
+						$e->getMessage()
+					);
+
+					$this->save_error( $plugin_slug, $message );
+				}
+
+				// Discard feedback.
+				ob_end_clean();
+			}
+
+			wp_clean_plugins_cache();
+
+			// Activate this thing.
+			if ( $activate ) {
+				try {
+					$result = activate_plugin( $installed ? $installed_plugins[ $plugin_file ] : $plugin_slug . '/' . $plugin_file );
+
+					if ( is_wp_error( $result ) ) {
+						throw new Exception( $this->get_error_message( $result ) );
+					}
+				} catch ( Exception $e ) {
+					$error   = true;
+					$message = sprintf(
+						// translators: 1: plugin name.
```

Comment:
The second variable is not defined for translators.

Note that the format for this is usually something like
```
// translators: Placeholders %1$s and %2$s are opening and closing <em> tags.
```

Reply:
Good catch! Updated here: https://github.com/Automattic/sensei/pull/3190/commits/3e215db7ddee6932804546252736b183cb9d0d75